rxbinding/src/main/java/com/jakewharton/rxbinding/view/ObservableGestureListener.java

```diff
@@ -0,0 +1,110 @@
+package com.jakewharton.rxbinding.view;
+
+import android.view.GestureDetector;
+import android.view.MotionEvent;
+import android.view.View;
+
+import rx.Observable;
+import rx.subjects.PublishSubject;
+
+final class ObservableGestureListener implements GestureDetector.OnGestureListener {
+
+    final View view;
+
+    private PublishSubject<ViewGestureEvent> downGestureObservable;
+    private PublishSubject<ViewGestureEvent> showPressGestureObservable;
+    private PublishSubject<ViewGestureEvent> singleTapUpGestureObservable;
+    private PublishSubject<ViewGestureScrollEvent> scrollGestureObservable;
+    private PublishSubject<ViewGestureEvent> longPressGestureObservable;
+    private PublishSubject<ViewGestureFlingEvent> flingGestureObservable;
```

Comment:
All these makes me _super_ nervous. I'd rather use a single `PublishSubject` and then just use `ofType` to filter for the convenience methods.